.github/workflows/ci.yml

```diff
@@ -400,16 +400,11 @@ jobs:
   create-tarball:
     name: Create source tarball
     if: contains(github.head_ref, 'dist') || contains(github.head_ref, 'extras') || contains(github.ref, 'release-')
-    needs: [build-and-test]
```

Comment:
I think this is important, because we want everything to be green before going ahead with any other steps, no?

Reply:
Of course, I will return it back once PR is finished. It's just to speed up the testing.